portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,19 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+    <!DOCTYPE html>
```

Comment:
Nit: It looks like the whole file has had an extra level of indentation added, which I think should be removed.

Reply:
To clarify, there should be two spaces of indentation.